src/scripts/choices.js

```diff
@@ -219,12 +219,13 @@ class Choices {
     this.passedElement.reveal();
     this.containerOuter.unwrap(this.passedElement.element);
 
+    this.clearStore();
+
     if (this._isSelectElement) {
       this.passedElement.options = this._presetChoices;
     }
 
-    this.clearStore();
-
+    this._presetChoices = [].concat(this.config.choices);
```

Comment:
Is `this._presetChoices` modified? Not sure why this is needed

Reply:
It was [here](https://github.com/jshjohnson/Choices/blob/0829899284f7a45a9d9ee9cc3902fb14cbfd1bf8/src/scripts/choices.js#L1905-L1917).
I need this change to prepare for the next `init` call. But this change is incomplete.
`this._presetChoices` contains options those are built from `this.config.choices`. They should not restore to `passedElement.options`.

I will fix it with next commit.

Reply:
I commited 79a4ae0 . It solve the above issue.
Coping `this.config.choices` is necessary. It avoids appending duplicated items on second `init`.